lib/pavilion/plugins/results/table.py

```diff
@@ -0,0 +1,98 @@
+from pavilion import result_parsers
+import yaml_config as yc
+import re
+from pavilion.utils import dbg_print
+
+class Table(result_parsers.ResultParser):
+
+    """Parses tables."""
+
+    def __init__(self):
+        super().__init__(
+            name='table',
+            description="Parses tables"
+        )
+
+    def get_config_items(self):
+
+        config_items = super().get_config_items()
+        config_items.extend([
+            yc.StrElem(
+                'delimiter', required=True,
+                help_text="Constant that will be placed in result."
+            ),
+            yc.StrElem(
+                'col_num', required=True,
```

Comment:
Should default to the number of column names if those are given.

Reply:
I check this logic in `_check_args`